invenio_app_ils/ui/src/invenio_app_ils/routes/urls.js

```diff
@@ -110,3 +110,13 @@ export const BackOfficeRoutes = {
   ...BackOfficeRoutesList,
   ...BackOfficeRouteGenerators,
 };
+
+const documentDetailsRecordViewStats = `/documents/:documentPid/record-stats`;
+
+export const DocumentApis = {
```

Comment:
Sorry, I was not clear, I meant to add the action in the `document APIs` as for all other `REST` calls:
https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/ui/src/invenio_app_ils/common/api/documents/document.js#L180